protoc.go

```diff
@@ -53,9 +55,16 @@ func (p *protocCmd) run() error {
 		log.Fatalln(err)
 	}
 
+	shcmd := "sh"
+	sharg := "-c"
+	if runtime.GOOS == "windows" {
```

Comment:
Though I think technically it'll work, I agree this behavior (code smell) should be handled more explicitly, rather than at runtime. I'll think about a good solution.